bin/crystal

```diff
@@ -89,7 +89,7 @@ _canonicalize_file_path() {
     local dir file
     dir=$(dirname -- "$1")
     file=$(basename -- "$1")
-    (cd "$dir" 2>/dev/null && printf '%s/%s\n' "$(pwd -P)" "$file")
+    (cd "$dir" 2>/dev/null >/dev/null && printf '%s/%s\n' "$(pwd -P)" "$file")
```

Comment:
I set `CDPATH` on my system, which causes `cd` to output which directory it's cding to, because it can be ambiguous.